recipes/neovim/meta.yaml

```diff
@@ -0,0 +1,72 @@
+{% set name = "neovim" %}
+{% set version = "0.1.7" %}
+{% set sha256 = "d8f885d019b1ad608f36ae23b8f1b15b7e33585e16f7514666ab6c9809bb4b7e" %}
+
+package:
+  name: neovim
+  version: '0.1.7'
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://github.com/{{name}}/{{name}}/archive/v{{version}}.tar.gz
+  sha256: {{ sha256 }}
+  patches:
+      - Makefile.patch
+
+build:
+  number: 0
+  skip: True  #[win]
+  rpaths:
+    - lib/
+
+requirements:
+  build:
+      - gcc # [linux]
+      - llvm # [osx]
+      - zlib
+      - openssl
+      - curl
+      - cmake
+      - m4
+      - libtool
+      - automake
+      - autoconf
+      - libiconv
+      - readline
+      - ncurses
+      - unzip
+      - pkgconfig
```

Comment:
Probably want `pkg-config` (note `-`).

(The other is an unfortunately named Python package. 😒)